exercises/php/trivia/Game.php

```diff
@@ -144,38 +92,84 @@ function wasCorrectlyAnswered() {
 				if ($this->currentPlayer == count($this->players)) $this->currentPlayer = 0;
 				return true;
 			}
-
-
-
 		} else {
 
 			echoln("Answer was corrent!!!!");
-		$this->purses[$this->currentPlayer]++;
-			echoln($this->players[$this->currentPlayer]
-					. " now has "
-					.$this->purses[$this->currentPlayer]
-					. " Gold Coins.");
+			$this->getAnswerOutcome('correct');
```

Comment:
You could probably extract this and add it to a variable on the top called "correct" or "correctAnswer".